.pipelines/templates/soak-test.yaml

```diff
@@ -32,18 +28,20 @@ jobs:
 
         - script: |
             # Download kubeconfig for soak cluster
-            az login \
-              --service-principal \
-              -u=$(AAD_CLIENT_ID) \
-              -p=$(AAD_CLIENT_SECRET) \
-              -t=$(TENANT_ID) > /dev/null
+            az login -i > /dev/null
             az account set -s=$(SUBSCRIPTION_ID)
 
             az aks get-credentials \
               --resource-group $(CLUSTER_RESOURCE_GROUP) \
               --name $(CLUSTER_CONFIG)
           displayName: "Set KUBECONFIG"
 
+        - ${{ if eq(clusterConfig, 'csi-secrets-store-soak-linux-aks') }}:
+          - script: |
+              echo "Installing helm..."
+              curl https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash
```

Comment:
Can we use the existing make target for installing helm?

Reply:
updated

Reply:
Tests are [passing](https://dev.azure.com/AzureContainerUpstream/Secrets%20Store%20CSI%20Driver%20Provider%20Azure/_build/results?buildId=27052&view=results).